components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -129,9 +130,9 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
          */
 
         String redirectURL;
-        Cookie opBrowserStateCookie = OIDCSessionManagementUtil.getOPBrowserStateCookie(request);
+        String opBrowserStateCookieValue = getOPBSCookieValue(request);
```

Comment:
I prefer to rename this something like opBrowserState?
obps cookie was one way to track the opBrowserState. And in the subsequent requests where the cookie is not present we are actually retrieving the same opBrowserState using some other means.

Reply:
+1. Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1519/commits/dda7bf80cc7f604b87669ae653ed1ac12089cbb2.